sass/components/mixins/page.scss

```diff
@@ -0,0 +1,19 @@
+$default-page-width: 50%;
```

Comment:
please do not define variables in this file. variables only allowed in variables.scss